codec/common/src/cpu.cpp

```diff
@@ -359,6 +359,32 @@ uint32_t WelsCPUFeatureDetect (int32_t* pNumberOfLogicProcessors) {
     return get_cpu_flags_from_cpuinfo();
 }
 
+#elif defined(__loongarch__) && defined(__linux__)
+/* for loongson */
```

Comment:
adjust the comment in line 320 and here.

Reply:
Thanks, will update.